docs/content/install/install-on-kubernetes.md

```diff
@@ -91,7 +91,7 @@ This will deploy a single Athens instance in the `default` namespace with `disk`
 
 ### Give Athens access to private repositories via Github Token (Optional)
 1. Create a token at https://github.com/settings/tokens
-2. Provide the token to the Athens proxy either through the [config.toml](https://github.com/gomods/athens/blob/master/config.dev.toml#L111) file or by setting the `ATHENS_GITHUB_TOKEN` environment variable.
+2. Provide the token to the Athens proxy either through the [config.toml](https://github.com/gomods/athens/blob/master/config-example.toml#L111) file or by setting the `ATHENS_GITHUB_TOKEN` environment variable.
```

Comment:
This line number is too specific. I would rephrase it in this or another PR a that is not the focus of the PR

Reply:
I agree that line numbers shouldn't be included in the docs. Did however not change it here since it isn't the scope for this PR.

Reply:
@manugupt1 I'll open up an issue (and a PR for it) to remove the line number, so we can go forward with this PR

Reply:
https://github.com/gomods/athens/issues/1157